solutions/observability/apm/resource-attributes.md

```diff
@@ -36,13 +40,63 @@ processors:
 ...
 ```
 
-::::{tip}
-Need to add event attributes instead? Use attributes—​not to be confused with resource attributes—​to add data to span, log, or metric events. Attributes can be added as a part of the OpenTelemetry instrumentation process or with the [attributes processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor).
+## Handling of unmapped attributes
+
+Only a subset of OpenTelemetry resource attributes are directly mapped to ECS fields. If an attribute doesn't have a predefined ECS mapping, the systems stores it under `labels.*`, with dots replaced by underscores.
+
+For example, if an OpenTelemetry resource contains:
+
+```json
+{
+  "service.name": "user-service",
+  "deployment.environment": "production",
+  "otel.library.name": "my-lib",
+  "custom.attribute.with.dots": "value"
+}
+```
+
+Elastic APM stores the following:
 
-::::
+```json
+{
+  "service.name": "user-service",
+  "service.environment": "production",
+  "labels": {
+    "otel_library_name": "my-lib",
+    "custom_attribute_with_dots": "value"
+  }
+}
+```
 
-% Stateful only after this?
+## Scope attributes translation
+
+Scope attributes are translated as follows:
+
+| OpenTelemetry attribute | Elastic APM field |
+|-------------------------|-------------------|
+| scope.name | service.framework.name |
+| scope.version | service.framework.version |
+
+## Conditional attributes translation
+
+Some OpenTelemetry attributes are conditionally converted based on their value type.
+
+Consider the following resource attributes:
+
+```json
+{
+  "http.status_code": 200,
+  "feature.enabled": true
+}
+```
 
-Elastic integrates with OpenTelemetry, allowing you to reuse your existing instrumentation to easily send observability data to the {{stack}}.
+The previous resource attributes are stored by Elastic APM as follows:
 
-For more information on how to combine Elastic and OpenTelemetry, see [OpenTelemetry integration](/solutions/observability/apm/use-opentelemetry-with-apm.md).
\ No newline at end of file
+```json
+{
+  "http.response.status_code": 200,
+  "labels": {
+    "feature_enabled": true
+  }
+}
+```
```

Comment:
I don't understand this section. How is it different from the section above?

Reply:
The community contributor thought it relevant to explain that some fields were conditionally created. Could we just merge both?

Reply:
There are some conditionals in the code, but the example here provides no value. I think we can just remove this section altogether.